Update issue templates (#8939)
kyma-project/kyma: .github/ISSUE_TEMPLATE/support_request.md

---
name: Support request
about: Request support or ask a question relating to Kyma
labels: support
---

<!-- Thank you for your contribution. Before you submit the issue:
1. Search open and closed issues for duplicates.
2. Read the contributing guidelines.
-->

**Description**

<!-- Provide a clear and concise description of the problem.
Describe where it appears, when it occurred, and what it affects.  -->

<!-- Provide relevant technical details such as the Kubernetes version, the cluster name and provider, the Kyma version, the browser name and version, or the operating system. -->

**Expected result**

<!-- Describe what you expect to happen. -->

**Actual result**

<!-- Describe what happens instead. -->

**Steps to reproduce**

> <!-- List the steps to follow to reproduce the bug. Attach any files, links, code samples, or screenshots that could help in investigating the problem. -->

Reviewer: A small suggestion that could help in distinguishing this template from a bug template 
```suggestion
<!-- Attach any files, links, code samples, or screenshots that are relevant. -->
```
